### PR TITLE
ECP-2415 -- add extra details to charge request

### PR DESCRIPTION
**Pull request by nicmollel-ro**
WHAT:
- https://ro-engine.atlassian.net/browse/ECP-2415
- adds payment identifier and member identifier to charge metatada in stripe

### CODE REVIEW

File: aa_stripe/models.py [hunk 1]
```diff
@@ -409,8 +409,15 @@ def charge(self, idempotency_key=None):
             metadata = {
                 "object_id": self.object_id,
                 "content_type_id": self.content_type_id,
+                "member_uuid": str(self.user.uuid),
```
Comment (remik) [inline aa_stripe/models.py:412]: This lib is public and some other users might not have this field
```suggestion
                "member_uuid": str(self.user.uuid) if hasattr(self.user, "uuid") else None,
```

File: aa_stripe/models.py [hunk 2]
```diff
@@ -409,8 +409,15 @@ def charge(self, idempotency_key=None):
...
+                "origin": "roman_api",
```
Comment (remik) [inline aa_stripe/models.py:413]: Please move it to some settings.

**Review by remik — CHANGES_REQUESTED** [review]

File: aa_stripe/models.py (hunk 1 above)
Comment (nicmollel-ro) [reply aa_stripe/models.py]: made the updates

**Review by remik — APPROVED** [review]
Looks good, do you want to bump version and changelog in the same PR?

**Comment by nicmollel-ro** [comment]
@remik i updated release details and version. if you can review

**Review by remik — APPROVED** [review]

**Comment by remik** [comment]
Looks good. Let me know if you have access with releasing it. If not I can add you to pypi or just release.

**Pull request merged**